synapse/cores/common.py

```diff
@@ -2005,6 +2005,10 @@ def formTufoByProp(self, prop, valu, **props):
         if ctor is not None:
             return ctor(prop, valu, **props)
 
+        tname = self.getPropTypeName(prop)
```

Comment:
Content of this change looks good once other unit tests are fixed. Can remove the additional call to `self.getPropTypeName` below.